infer/environment.cc

```diff
@@ -1011,7 +1011,8 @@ core::TypePtr Environment::processBinding(core::Context ctx, const cfg::CFG &inW
                     core::lsp::QueryResponse::pushQueryResponse(
                         ctx, core::lsp::SendResponse(core::Loc(ctx.file, bind.loc), retainedResult, send->fun,
                                                      send->isPrivateOk, ctx.owner.asMethodRef(),
-                                                     core::Loc(ctx.file, send->receiverLoc)));
+                                                     core::Loc(ctx.file, send->receiverLoc),
+                                                     static_cast<u2>(dispatchArgs.args.size())));
```

Comment:
Thanks for refactoring! One of the reasons why I suggested this refactoring is because the number of args actually has nothing to do with the dispatch itself, but is a property of the ast / cfg itself:

```suggestion
                                                     static_cast<u2>(send->args.size())));
```

this refactoring should make that more clear.